cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -349,7 +353,7 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         })) {
           statsOnlyResultTable()
         }
-        p("The resulting role should have all privileges to manage roles:")
+        p("The resulting role should have all privileges to manage roles (except showing privileges):")
```

Comment:
Maybe take this out. If our line is that managing roles is not the same thing as managing privileges this may confuse that.

Reply:
You are right! I'll remove that.